src/ColorTypes.jl

```diff
@@ -50,7 +50,7 @@ include("show.jl")
 end # module
 
 if VERSION < v"0.4.0-dev"
-    using Docile
+    using Docile, Compat
```

Comment:
I noticed this line change is totally unnecessary now. You can revert to the old code.
This happened after Compat had a version update a few hours ago. 
https://github.com/JuliaLang/METADATA.jl/commit/f38bcc2c6b1feab55b010bb7b7a322e9fb52cd6a